pilot/cmd/pilot-discovery/main.go

```diff
@@ -154,6 +158,11 @@ func init() {
 		"File containing the x509 Server Certificate")
 	discoveryCmd.PersistentFlags().StringVar(&serverArgs.ServerOptions.TLSOptions.KeyFile, "tlsKeyFile", "",
 		"File containing the x509 private key matching --tlsCertFile")
+	discoveryCmd.PersistentFlags().StringSliceVar(&serverArgs.ServerOptions.TLSOptions.TLSCipherSuites, "tls-cipher-suites", nil,
+		"Comma-separated list of cipher suites for the server. "+
```

Comment:
For better code readability, please change "Comma-separated list of cipher suites for the server" to "Comma-separated list of cipher suites for the istiod TLS server".
In addition, please add an example configuration for tls-cipher-suites.

Reply:
If user run `cmd -h`. It will tell him how to configure.